src/styles/index.scss

```diff
@@ -1,3 +1,152 @@
+@font-face {
+  font-family: Roboto-Regular;
+  src: url(../fonts/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: Roboto-Bold;
+  src: url(../fonts/Roboto-Bold.ttf);
+}
+
+@font-face {
+  font-family: Roboto-medium;
+  src: url(../fonts/Roboto-Medium.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Light.ttf);
+}
+
+@import '../utils/mixins';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background-color: $button_color_hover;
+  border: 1px solid $border_color;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
```suggestion
```
It is common practice to apply `box-sizing: border-box;` for all elements via `*` selector